### ISSUE TITLE
Error in example of learning rate finder docs 

### ISSUE THREAD

**Issue by pbmstrk** [PyTorchLightning/pytorch-lightning#4103]
## 📚 Documentation

There seems to be a minor error in the [docs for the learning rate finder functionality](https://pytorch-lightning.readthedocs.io/en/latest/lr_finder.html) of the trainer module. In particular, the example uses `trainer.lr_find`, however this causes an AttributeError, it should be `trainer.tuner.lr_find`.

In addition, the page contains two nearly identical examples - one written in the [doc file](https://github.com/PyTorchLightning/pytorch-lightning/blame/master/docs/source/lr_finder.rst#L82-L102) and one generated from the docstring of the [function](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/tuner/lr_finder.py#L114-L130) - not sure if this is intended.
